controller/search.go

```diff
@@ -129,3 +131,65 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		return ctx.OK(&response)
 	})
 }
+
+// Users runs the user search action.
+func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
+
+	q := ctx.Q
+	if q == "" {
+		return ctx.BadRequest(goa.ErrBadRequest(fmt.Errorf("empty search query not allowed")))
+	} else if q == "*" {
+		q = "" // Allow empty query if * specified
+	}
+
+	var result []account.Identity
+	var count int
+	var err error
+
+	offset, limit := computePagingLimts(ctx.PageOffset, ctx.PageLimit)
```

Comment:
There are many places where we use this method with a typo in the name - please allow me to update the same in a different PR :)

```
controller/collaborators.go:52: undefined: computePagingLimts
controller/namedspaces.go:59: undefined: computePagingLimts
controller/planner_backlog.go:45: undefined: computePagingLimts
controller/search.go:43: undefined: computePagingLimts
controller/search.go:98: undefined: computePagingLimts
controller/search.go:149: undefined: computePagingLimts
controller/space.go:196: undefined: computePagingLimts
controller/space_codebases.go:76: undefined: computePagingLimts
controller/work_item_comments.go:73: undefined: computePagingLimts
controller/work_item_comments.go:99: undefined: computePagingLimts
controller/work_item_comments.go:99: too many errors
```